Source/Other/Utility/SPFunctions.m

```diff
@@ -159,3 +159,18 @@ id DumpObjCMethods(Class clz) {
     return arr;
 }
 
+NSInteger intSort(id num1, id num2, void *context)
+{
+    // JCS not: I want descending, so I've swapped the return values
```

Comment:
should we name the functions `desc`?

Reply:
ok, it is the opposite of what is expected! thnks